arch/arm/src/Makefile

```diff
@@ -35,7 +35,7 @@
 ############################################################################
 
 include $(TOPDIR)/Make.defs
-include chip/Make.defs
+-include chip/Make.defs
```

Comment:
why change? I think chip/Make.defs isn't an optional file.

Reply:
As I said above `clean_context` is used before linking the directories, at that point chip/ doesn't exist.

Reply:
Ok, understand!